src/utils/credentials.ts

```diff
@@ -1,6 +1,7 @@
 import { writeFile } from 'fs/promises';
 import { readFile } from 'fs/promises';
 import { DB, Credential } from '../types';
+import { encryptCredential, decryptCredential } from './crypto';
 
 export async function readCredentials(): Promise<Credential[]> {
```

Comment:
This one is used in one of your routes, but will not decrypt your passwords. You should implement this feature at some point. It is a medium sized change so I recommend implementing it after we've talked about MongoDB